### PR TITLE
Creating addons

### PR DESCRIPTION
**Pull request by winkerVSbecks**
To Do:

- [x]  `track-state.md` add an image of the "payoff", a gif that shows a closeup of the tool button being clicked/unclicked and changing state.
- [x] Make the outline CSS more compact or link to a GitHub file.
- [x] `decorators.md` add payoff image of what the addon should now look like
- [x] Add a `conclusion.md`:
  * Concludes the guide
  * Learn more section that links to various addon resources, the Outline source code, and the demo Storybook to see it in action
  * Sign up to SB mailing list embed

### CODE REVIEW

**Comment by winkerVSbecks** [comment]
@jonniebigodes @domyen this is ready for feedback

File: content/creating-addons/react/en/track-state.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      key={TOOL_ID}
++      active={outlineActive}
+      title="Apply outlines to the preview"
++      onClick={toggleOutline}
+    >
+      <Icons icon="outline" />
+    </IconButton>
+  );
+};
+```
```
Comment (domyen) [inline content/creating-addons/react/en/track-state.md:56]: Add an image of the "payoff" – what the user is supposed to see at this point. I'd suggest a gif that shows a closeup of the tool button being clicked/unclicked and changing state.

File: content/creating-addons/react/en/decorators.md [hunk 2]
```diff
@@ -0,0 +1,438 @@
...
+    style.setAttribute('id', selector);
+    style.innerHTML = css;
+    document.head.appendChild(style);
+  }
+};
+```
+
+And the outline CSS itself is based on what [Pesticide uses](https://github.com/mrmrs/pesticide/blob/master/css/pesticide.css).
+
+```js:title=src/outlineCSS.js
```
Comment (domyen) [inline content/creating-addons/react/en/decorators.md:134]: Can we make this CSS more compact? It's a lot of scrolling.

If not, I'd suggest we link to a GitHub file where they can download the CSS.

File: content/creating-addons/react/en/decorators.md [hunk 3]
```diff
@@ -0,0 +1,438 @@
...
+      outline: 1px solid #cc2643 !important;
+    }
+    ${selector} br {
+      outline: 1px solid #db687d !important;
+    }
+    ${selector} wbr {
+      outline: 1px solid #db175b !important;
+    }`;
+}
+```
```
Comment (domyen) [inline content/creating-addons/react/en/decorators.md:438]: Add payoff image of what the addon should now look like

File: content/creating-addons/react/en/add-to-catalog.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+Which will:
+
+- Build and package the addon code
+- Bump the version
+- Push a release to GitHub and npm
+- Push a changelog to GitHub
+
+There you have it! We’ve successfully published our package to npm and released our first Storybook Addon. There may be a delay in your addon showing up in the catalog because it crawls npm. If your addon isn’t showing up, please file an issue on the catalog repo.
+
+Addons help the community customize and automate Storybook. Publish your addon to share it with thousands of devs. I’m excited to see what you build!
```
Comment (domyen) [inline content/creating-addons/react/en/add-to-catalog.md:118]: Add a conclusion.md: 
- Concludes the guide
- **Learn more** section that links to various addon resources, the Outline source code, and the demo Storybook to see it in action
- **Sign up to SB mailing list** embed

**Review by domyen — CHANGES_REQUESTED** [review]

**Comment by domyen** [comment]
Adjust flex so that the grid is aligned left. I think this is just adjusting Guides_Content from `justify-content: center` to `justify-content: space-between`.
image

File: content/create-an-addon/react/en/preset.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+```js:title=src/preset/preview.js
+import { withGlobals } from '../withGlobals';
+
+export const decorators = [withGlobals];
+```
+
+<div class="aside"><b>Note:</b> the <code>withRoundTrip</code> decorator from the Addon Kit is an example of two-way communication between the story and an addon. However, we don't require that for our addon and can delete it.</div>
+
+Success! You now have a fully functional addon!
```
Comment (domyen) [inline content/create-an-addon/react/en/preset.md:26]:
```suggestion
Success! You now have a fully functional addon in your local Storybook. In the final chapter, learn how to list your addon in the catalog. That way you can share with your team and the Storybook community.
```

File: src/components/screens/GuideScreen/Hero.js [hunk 6]
```diff
@@ -161,6 +160,37 @@ const GuideImage = styled.img`
...
+  margin-left: -22px; // optical for rebalancing for centered layout. This accounts for the arrow
+
+  &:hover,
+  &:focus,
+  &:active {
+    color: ${color.lightest};
+  }
+
+  @media (min-width: ${breakpoint * 1.333}px) {
+    margin-top: 0;
```
Comment (domyen) [inline src/components/screens/GuideScreen/Hero.js:179]:
```suggestion
    margin-top: 0;
    margin-left: 0; // remove optical rebalancing of the arrow
```

**Comment by winkerVSbecks** [comment]
@domyen I updated the tweet message to use the sb.js.org url and reference the `storybookjs` twitter account

**Comment by domyen** [comment]
Are the deploy previews working?

**Comment by winkerVSbecks** [comment]
@domyen I renamed the netlify project so, I'll have to trigger them again. Will do with once I setup the learnstorybook.com redirect we spoke about.

**Comment by domyen** [comment]
lgtm! 

Chromatic "SB Publish" check on the source code repo isn't working for some reason (no project token?).

File: content/create-an-addon/index.md [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+  ]
+contributors: []
+twitterShareText: "I’m learning how to build a Storybook addon! They're great for customizing Storybook to fit your UI development workflow."
+---
+
+<h2>What you'll build</h2>
+
+<div class="badge-box">
+  <div class="badge">
+    <img src="/frameworks/logo-react.svg"> <a href="/create-an-addon/react/en/introduction/"> React</a>
```
Comment (domyen) [inline content/create-an-addon/index.md:40]: Let's remove the link here – we don't have consistent behavior for badge links which ends up being strange.

```suggestion
    <img src="/frameworks/logo-react.svg"> React
```

File: .vscode/settings.json [hunk 8]
```diff
@@ -1,3 +1,20 @@
 {
```
Comment (domyen) [inline .vscode/settings.json:1]: Did you mean to add your vscode settings?

Comment (winkerVSbecks) [reply .vscode/settings.json:1]: yea, I meant to save those as local settings. I removed them.

**Review by domyen — APPROVED** [review]

**Comment by domyen** [comment]
Hooray! Huge thanks Varun 🙏

**Pull request merged**